### ISSUE TITLE
Create paranoia feature

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kbjorgensen/kbsslenforcer#43]
```
It would be nice to have an option to disable all unencrypted requests and 
therefore enforce encryption on everything.

This is only really useful in situations where you can live without sites that 
don't support encryption and if you would really like to avoid using any 
unencrypted sites at all.

It's only really possible once issue 25 is implemented.

I suggest calling this the paranoia feature, in lack of a better description so 
far. Suggestions for names and feedback on the idea is very welcome.
```

Original issue reported on code.google.com by `k...@kbit.dk` on 22 Nov 2010 at 7:32
- Blocked on: #25
